tests/osaka/eip7692_eof_v1/eip3540_eof_v1/test_container_validation.py

```diff
@@ -935,9 +935,21 @@ def test_valid_containers(
             validity_error=[EOFException.MISSING_CODE_HEADER, EOFException.UNEXPECTED_HEADER_KIND],
         ),
         Container(
-            name="empty_type_section",
+            name="empty_type_section_empty_code",
             sections=[
-                Section(kind=SectionKind.TYPE, data="0x"),
+                Section(kind=SectionKind.TYPE),
+                Section.Code(),
+            ],
+            expected_bytecode="ef00 01 01 0000 02 0001 0000 04 0000 00",
```

Comment:
The only remaining case would be something like:
`ef00 01 01 0000 02 0000 04 0000 00`

Similar thing but zero code sections.

Reply:
I will add it, but I don't think there is a way to code it in without raw bytes...

Reply:
This test exist already as `"zero_code_sections_header_empty_type_section"`.